gips/data/core.py

```diff
@@ -274,6 +274,16 @@ def updated(self, newasset):
                 self._version < newasset._version)
 
 
+    def version_text(self):
```

Comment:
is this used?

Reply:
I just added it because I needed it for prism, and thought it would be generally useful to spit back a string that meant something in the context of the driver, instead of a meaningless number (which is sometimes just YYYYMMDD.HHMMSS or something even  less comprehensible).

Reply:
could be a garb-feat, but it makes sense at the moment.

Reply:
gotcha